### DISCUSSION TITLE
Release 2.3.0

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#910] [Announcements]
**New Features:**
- Support for WhiskyWine 2.0.0 with updated dependencies and GStreamer

**Quality of Life:**
- Added option to open a terminal window with the necessary environment variables set
- Significant improvements to locales

**Bug Fixes:**
- Fixed an issue where terminal commands were not properly escaped
- Fixed a visual issue with the height of the pin creation view
- Refactored installation checks, should result in less broken installs and a better WhiskyWine updater experience

**New Contributors:**
- @zorikon made their first contribution in https://github.com/Whisky-App/Whisky/pull/907
- @nastys made their first contribution in https://github.com/Whisky-App/Whisky/pull/843

**Full Changelog:** https://github.com/Whisky-App/Whisky/compare/v2.2.4...v2.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v2.3.0'>Release 2.3.0</a>.</em>

**Comment by realduug** [comment]
Hi, I’m looking for some guidance on setup and usage. I intend to use whisky to run trading platforms: cTrader & meta trader. Is .Net supported?
